modules/iframe-app/src/ConnextManager.tsx

```diff
@@ -223,17 +224,32 @@ export default class ConnextManager {
         const hasBalance = chan?.balances.find((balance) => {
           // this checks both alice and bob balance
           if (BigNumber.from(balance.amount[0]).gt(0) || BigNumber.from(balance.amount[1]).gt(0)) {
+            console.warn("Found channel with balance: ", chan);
             return true;
           }
           return false;
         });
-        if (hasBalance) {
+
+        const activeTransfers = await browserNode.getActiveTransfers({ channelAddress });
+        if (activeTransfers.isError) {
+          throw activeTransfers.getError();
+        }
+        const hasTransfers = activeTransfers.getValue().find((transfer) => {
+          if (BigNumber.from(transfer.balance.amount[0]).gt(0) || BigNumber.from(transfer.balance.amount[1]).gt(0)) {
+            console.warn("Found transfer with balance: ", chan);
+            return true;
+          }
+          return false;
+        });
+
+        if (hasBalance || hasTransfers) {
```

Comment:
we should add some comments on what it means to be `ableToMigrate` from a high level. I don't understand how it would be the case that if you have balance or transfers you *are* able to, while if you have an empty channel you *aren't* able to